requirements/extensions.txt

```diff
@@ -0,0 +1,8 @@
+# Iris extensions (i.e. key tools that depend on Iris)
+# ----------------------------------------------------
+
+# Note: pip can handle the circularity of these extensions, but conda will
+# struggle. To install these extensions, ensure iris[core] has been installed
+# first.
+
+iris_grib;python_version<"3"  #conda:
```

Comment:
@pelson the `extensions.txt` doesn't seem to be used anywhere. Intentional?

Reply:
Ja. It could happily live as an extra (e.g. ``pip install scitools-iris[extensions]``)...